config/settings/local.py

```diff
@@ -123,6 +123,16 @@
 CONNECTED_APP_CLIENT_SECRET = env("CONNECTED_APP_CLIENT_SECRET")
 CONNECTED_APP_CALLBACK_URL = env("CONNECTED_APP_CALLBACK_URL")
 
+METACI_APPS = {"local-app"}
```

Comment:
Is there ever a reason for this to be different from the keys of the AUTOSCALERS dict? Can we just use those keys to avoid the possibility of them being accidentally out of sync?